### PR TITLE
[PLAYER-3981] Update to Android 27 and Gradle 4.4

### PR DESCRIPTION
**Pull request by brol1dev**

### CODE REVIEW

File: vendor/GoogleCast/CastCompanionLibrary/build.gradle [hunk 1]
```diff
@@ -20,7 +20,7 @@ buildscript {
         google()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.0.1'
```
Comment (akshaydch11) [inline vendor/GoogleCast/CastCompanionLibrary/build.gradle]: Lets remove CastCompanionLibrary as we are not using it.

**Comment by akshaydch11** [comment]
Changed destination to candidate

File: vendor/GoogleCast/CastCompanionLibrary/build.gradle (hunk 1 above)
Comment (brol1dev) [reply vendor/GoogleCast/CastCompanionLibrary/build.gradle]: done

**Comment by brol1dev** [comment]
@akshaydch11 removed the CastCompanionLibrary, please review

**Comment by pilievOoyala** [comment]
LGTM.

**Pull request merged**